### PR TITLE
Search function.

### PR DESCRIPTION
**Pull request by elaquentil**

### CODE REVIEW

File: src/dropbox.ml [hunk 1]
```diff
@@ -183,6 +185,8 @@ module type S = sig
       contents: metadata list;
     }
 
+  type search = metadata list
```
Comment (Chris00) [inline src/dropbox.ml]: I think exporting this type only obscure the interface — one more indirection to remember...

**Comment by Chris00** [comment]
Please squash your commits.

**Comment by elaquentil** [comment]
I did some squash of my commits.
If the search function finds nothing with a query, it returns an empty metadata list.

**Comment by Chris00** [comment]
PR https://github.com/Chris00/ocaml-dropbox/pull/27 merged.

**Pull request closed without merging**